### ISSUE TITLE
Training nnunetv2 with train_loss and val_loss consistently negative

### ISSUE THREAD

**Issue by zgp123-wq** [MIC-DKFZ/nnUNet#1795]
I am attempting to train my own dataset using the nnunetv2 framework, but I have encountered an issue. During the training process, I have noticed that both train_loss and val_loss values consistently appear as negative numbers, which seems to be unexpected behavior. I am looking for a solution to address this problem.

**Comment by Lars-Kraemer** [comment]
Hey @zgp123-wq,

It's normal to encounter a negative loss during training in nnUNet. The Dice loss can be implemented either as "1 - dice" or as "-dice." In the first case, the optimal value is 0, while in the second case, it is -1. Since nnUNet combines the Dice loss (implemented as "-dice") and Cross Entropy loss (with the best value being 0), the ideal loss value in nnUNet is -1.

Best regards,
Lars
